api/views.py

```diff
@@ -36,3 +43,33 @@ class TraineeQuizzesView(generics.ListAPIView):
     serializer_class = TraineeQuizzesSerializer
     filter_fields = ('title', 'description', 'type_of_quiz')
     search_fields = ('title', 'description', 'type_of_quiz')
+
+
+class CountryIndicatorViewSet(viewsets.ModelViewSet):
+    queryset = CountryIndicator.objects.all()
+    serializer_class = CountryIndicatorSerializer
+    filter_fields = (
+        'iso_code', 'name', 'pdi', 'idv', 'mas', 'uai', 'lto', 'ivr')
+    search_fields = (
+        'iso_code', 'name', 'pdi', 'idv', 'mas', 'uai', 'lto', 'ivr')
```

Comment:
In my opinion filtering by indicators and searching by iso_code and name will be enough.